modules/manage/pages/security/authentication.adoc

```diff
@@ -260,7 +260,184 @@ rpk cluster config set sasl_mechanisms '["SCRAM","GSSAPI"]'
 rpk cluster config set kafka_enable_authorization true
 ----
 
-NOTE: If you're configuring authentication for the first time, you may need to configure xref:./authorization.adoc#acls[ACLs] before users can access Redpanda resources.
+NOTE: If you're configuring authentication for the first time, you may need to 
+configure xref:./authorization.adoc#acls[ACLs] before users can access Redpanda resources.
+
+=== Enable OIDC
```

Comment:
Not sure why you are pulling in these other topics @greg-szrama . Please take a look--also asked @Deflaimun to take a look. Not reviewing these extraneous topics in this PR.

Reply:
I didn't edit these, I'm not sure why they're in the change list. I did reach out to @Deflaimun  on them.